pkg/runner/run_context.go

```diff
@@ -480,11 +480,13 @@ func (rc *RunContext) platformImage(ctx context.Context) string {
 func (rc *RunContext) options(ctx context.Context) string {
 	job := rc.Run.Job()
 	c := job.Container()
-	if c == nil {
-		return rc.Config.ContainerOptions
+	containerOptions := ""
+	if c != nil {
+		containerOptions = c.Options
 	}
+	containerOptions += " " + rc.Config.ContainerOptions
```

Comment:
I don't really think it's a good idea. What do we expect Docker to do when the result is `"--network bridge" + " " + "--network host" `?